code/game/dna/genes/vg_powers.dm

```diff
@@ -40,6 +40,7 @@ Obviously, requires DNA2.
 			M.update_mutations()		//update our mutation overlays
 			M.update_body()
 			to_chat(M, "<span class='warning'>You suddenly feel very weak.</span>")
+			M.species.muscle_mass = 150
```

Comment:
You have two options

>Hulk adds/subtracts a fixed amount
>Hulk saves your starting muscle mass so that you get it back later

You can make magic numbers into defines and PJB won't yell at you